STL_Extension/include/CGAL/hash_openmesh.h

```diff
@@ -37,6 +37,11 @@ class OMesh_edge {
     }
   }
 
+/*
+  operator OpenMesh::EdgeHandle () const {
+    return OpenMesh::EdgeHandle(idx());
+  }
+*/
```

Comment:
is this needed?

Reply:
Should go to another pull request.  The thing is that the `edge_descriptor` we wrote cannot be passed to methods like `status(e)`.